src/hashtable.py

```diff
@@ -84,7 +133,20 @@ def resize(self):
 
         Fill this in.
         '''
-        pass
+        #Create new Hashtable
+        ht = HashTable(self.capacity*2)
+
+        #Copy via loop
```

Comment:
Good implementation here. However, we can use the same `HashTable` instance and call `self.capacity * 2` and so on without having to create a new instance. Logic is spot on here though.